mods/dungeon_loot/mapgen.lua

```diff
@@ -0,0 +1,168 @@
+minetest.set_gen_notify({dungeon = true, temple = true})
+
+local function noise3d_integer(noise, pos)
+	return math.abs(math.floor(noise:get3d(pos) * 2147483647))
```

Comment:
`* 0x7FFFFFFF` would be more readable IMO

Reply:
Accept.